web/wdio.conf.ts

```diff
@@ -80,6 +86,20 @@ export const config: WebdriverIO.Config = {
                     ...(userConfig?.plugins ?? []),
                     tsconfigPaths(),
                 ],
+                resolve: {
```

Comment:
Big fix: Fixed the Alias pathing for Vite, which is what Wdio uses for unit testing.